chart/values.yaml

```diff
@@ -0,0 +1,82 @@
+# Default values for standards-insights.
+
+image:
+  repository: public.ecr.aws/g1r8z6f4/standards-insights
+  pullPolicy: IfNotPresent
+  tag: "" # Defined by chart appVersion parameter
+
+# Define environment variables
+env: {}
+envFrom: {}
+
+logLevel: info
+logFormat: json
+configFilePath: "/config/config.yaml"
+
+# Set additional labels on all resources
+additionalLabels: {}
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+volumeMounts: {}
+volumes: {}
+
+persistentVolumeClaim:
+  create: true
+  annotations: {}
+  name: ""
+  storage: "50Gi"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext:
+  fsGroup: 10001
+
+securityContext:
+  capabilities:
+    drop:
+    - ALL
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsGroup: 10001
+  runAsUser: 10001
+
+service:
+  type: ClusterIP
+  port: 3000
+
+serviceMonitor:
+  enabled: true
+  interval: 60s
+  scrapeTimeout: 10s
+  additionalLabels: {}
+
+resources:
```

Comment:
Do not set resources by default because it makes it hard to override later on for end users. It's common practise to comment this part and let users decide what to use. We can set recommendation in the commented resources.